pkg/cmd/admin/policy/modify_roles.go

```diff
@@ -388,7 +405,7 @@ subjectCheck:
 }
 
 func (o *RoleModificationOptions) RemoveRole() error {
-	roleBindings, err := o.RoleBindingAccessor.GetExistingRoleBindingsForRole(o.RoleNamespace, o.RoleName)
+	roleBindings, err := o.RoleBindingAccessor.GetExistingRoleBindingsForRole(o.RoleNamespace, o.RoleName, o.RoleBindingName)
```

Comment:
Specifying `RoleBindingName` here seems strange to me.  If you are removing a role, you probably want to remove it from all bindings.  Otherwise you risk the user continuing to have the permission you just tried to remove.